packages/logging/amplify_logging_cloudwatchlogs_dart/pubspec.yaml

```
@@ -0,0 +1,16 @@
+name: amplify_logging_cloudwatchlogs_dart
+description: Dart implementation of the Amplify Logging plugin using CloudWatchLogs.
+version: 0.1.0
+homepage: https://docs.amplify.aws/lib/q/platform/flutter/
+repository: https://github.com/aws-amplify/amplify-flutter/tree/next/packages/logging/amplify_logging_cloudwatchlogs_dart
+issue_tracker: https://github.com/aws-amplify/amplify-flutter/issues
+
+environment:
+  sdk: '>=2.18.4 <3.0.0'
```

`2.17.0` is the min SDK version for the repo right now

```suggestion
  sdk: '>=2.17.0 <3.0.0'
```